src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/AppointmentsServiceTest.kt

```diff
@@ -0,0 +1,77 @@
+package uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.service
+
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data.CreateAppointment
+import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data.CreateAppointmentAddress
+import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.jpa.entity.Category
+import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.jpa.repository.PrisonerRepository
+import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.service.external.ResettlementPassportDeliusApiService
+import java.time.LocalDateTime
+
+@ExtendWith(MockitoExtension::class)
+class AppointmentsServiceTest {
```

Comment:
This test is good but could you add some Integration tests as well as each response code?